golem/environments/environment.py

```diff
@@ -37,6 +37,13 @@ def ok(cls) -> 'SupportStatus':
     def err(cls, desc) -> 'SupportStatus':
         return cls(False, desc)
 
+    @property
+    def err_reason(self):
+        try:
+            return list(self.desc.keys())[0]

```

Comment:
```suggestion
            return next(iter(self.desc))
```
You'd need to catch a `StopIteration` exception though.